pkg/api/azenvtypes.go

```diff
@@ -64,6 +65,18 @@ type AzureOSImageConfig struct {
 	ImageVersion   string `json:"imageVersion,omitempty"`
 }
 
+// AzureTelemetryPID represents the current telemetry ID
+type AzureTelemetryPID string
+
+const (
+	// DefaultAzurestackDeployTelemetryPID tracking ID for Deployment
+	DefaultAzurestackDeployTelemetryPID = "pid-1bda96ec-adf4-4eea-bb9a-8462de5475c0"
```

Comment:
Can we have AzureStack rather Azurestack in all the variable name.

Reply:
Done